connectors/src/lib/models/microsoft.ts

```diff
@@ -0,0 +1,50 @@
+import type {
+  CreationOptional,
+  ForeignKey,
+  InferAttributes,
+  InferCreationAttributes,
+} from "sequelize";
+import { DataTypes, Model } from "sequelize";
+
+import { sequelizeConnection } from "@connectors/resources/storage";
+import { ConnectorModel } from "@connectors/resources/storage/models/connector_model";
+
+export class MicrosoftConfiguration extends Model<
+  InferAttributes<MicrosoftConfiguration>,
+  InferCreationAttributes<MicrosoftConfiguration>
+> {
+  declare id: CreationOptional<number>;
+  declare createdAt: CreationOptional<Date>;
+  declare updatedAt: CreationOptional<Date>;
+  declare connectorId: ForeignKey<ConnectorModel["id"]>;
+}
+MicrosoftConfiguration.init(
```

Comment:
Will this be shared between teams and sharepoint?

Reply:
We may actually need 2 different objects, it would be easier if we have different settings (index pdf will not be useful for teams)